### PR TITLE
497 gif images

### PR DESCRIPTION
**Pull request by HarshaVardhanNakkina**
add gif images to the reasons
- added a separate file for reasons with gifs because I thought it'll be easier to verify, you can delete the original file while merging, if you like the changes
- selects a random gif URL from a pre-defined array of URLs
- displays gif below the reason
- and some style changes to make the gif look good
- fixed a file name typo
image

### CODE REVIEW

**Comment by HarshaVardhanNakkina** [comment]
tenor images seem to be faster than giphy, we might want to replace giphy URLs with tenor ones

File: component/widget.js [hunk 1]
```diff
@@ -60,13 +60,17 @@ export default class Widget extends React.Component {
...
       <div className="item">
         <h3 className="tagline">Should I Deploy Today?</h3>
-        <h2 id="text">{this.state.reason}</h2>
-        <span id="reload" onClick={this.onSpacePressOrClick}>
+        <div id="reload" onClick={this.onSpacePressOrClick}>
           Hit <span className="space-btn">Space</span> or Click
-        </span>
+        </div>
+        <h2 id="text">{reasonText}</h2>
+        <img className="gif-reason" src={gifUrl} alt={reasonText} />
```
Comment (baires) [inline component/widget.js:73]: We can use `next/image` here to have Image Optimization out of the box

File: pages/api/index.js [hunk 2]
```diff
@@ -18,6 +18,6 @@ export default (req, res) => {
   res.status(200).json({
     timezone: timezone,
     shouldideploy: shouldIDeploy(time),
-    message: getRandom(dayHelper(time))
+    message: getRandom(dayHelper(time)).reason
```
Comment (baires) [inline pages/api/index.js:21]: We can include the gif as well here

File: pages/api/slack/index.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ export default (req, res) => {
     attachments: [
       {
         text: time
-          ? getRandom(dayHelper(time))
+          ? getRandom(dayHelper(time)).reason
```
Comment (baires) [inline pages/api/slack/index.js:19]: Same as before

**Review by baires — CHANGES_REQUESTED** [review]

**Comment by baires** [comment]
I really appreciate the hard work here. Im gonna keep this PR open for now I want to re-think a little more the whole idea/layout now that you include images.

**Pull request closed without merging**